docs/book/toc.md

```diff
@@ -27,6 +27,8 @@
   * [Upgrade your ZenML server](how-to/manage-zenml-server/upgrade-zenml-server.md)
   * [Best practices for upgrading ZenML](how-to/manage-zenml-server/best-practices-upgrading-zenml.md)
   * [Using ZenML server in production](how-to/manage-zenml-server/using-zenml-server-in-prod.md)
+  * [Use the ZenML VSCode extension](how-to/manage-zenml-server/vscode-extension.md)
+  * [Chat with your ZenML server](how-to/manage-zenml-server/mcp-chat-with-server.md)
```

Comment:
Not usre this is the right place for it tbh... but as we're restructuring anyway I guess it doesn't matter?

Reply:
It was the place that seemed to make the most sense. With both MCP and VSCode you're administering your server, right? But yeah it's all going to change anyway.